api/workAllocation/util.ts

```diff
@@ -88,24 +90,45 @@ function getAssigneeName(task: any, caseworkers: Caseworker[]): string {
   return null
 }
 
-export function mapCaseworkerData(caseWorkerData: any[]): any[] {
+export function mapCaseworkerData(caseWorkerData: CaseworkerApi[]): Caseworker[] {
+  const caseworkers: Caseworker[] = []
```

Comment:
Just a suggestion here. We could use a map `caseWorkerData.map(caseWork => ({...mappedObject}))`

Reply:
Agreed. I believe it is nice to have